www/src/components/blog-post-preview-item.js

```diff
@@ -25,7 +25,21 @@ class BlogPostPreviewItem extends React.Component {
             marginBottom: rhythm(2),
           }}
         >
-          <Img
+          <Link
+            to={post.frontmatter.author.fields.slug}
+            css={{
+              boxShadow: `none !important`,
+              borderBottom: `0 !important`,
```

Comment:
You could move the keys (here and also the ones below) that require `!important` to the `&&` selector which glamor offers to increase specifity (ref. https://github.com/threepointone/glamor/blob/master/docs/selectors.md).

Reply:
Seems like I should do the same here then?
https://github.com/gatsbyjs/gatsby/pull/5813/files/b9c097e3eba19af0a824098aefe1e812ed00125b#diff-3f9ee2e9c0e318b6189a8f786092c9f5R164

`template-blog-post.js`

L164
```jsx
color: `${colors.gatsby} !important`,
```

Reply:
I went ahead and committed a change with suggestion above. Changed my mind about the [last comment](https://github.com/gatsbyjs/gatsby/pull/5813#discussion_r196211494) though, and maybe the commit. Color me puzzled. Perhaps I am not understanding Glamor but I do not see how given HTML like

```html
<a class="css-9rkwk9" href="/contributors/michelle-barker/">Michelle Barker</a>
```

the resulting CSS:

```css
.css-9rkwk9.css-9rkwk9, [data-css-9rkwk9][data-css-9rkwk9] {
	color: #663399 !important;
	font-weight: normal;
}
```

is more specific than

```css
.css-9rkwk9,[data-css-9rkwk9]{
	position:relative;
	z-index:1;
}
```

`.css-9rkwk9.css-9rkwk9` seems redundant to me. If it is I can revert the last commit. If it makes sense to you then this is ready unless you want me to wrap

```jsx
color: `${colors.gatsby} !important`,
```

in an `&&` section in `template-blog-post.js` as well [per above](https://github.com/gatsbyjs/gatsby/pull/5813#discussion_r196211494).

Reply:
The chained selectors (e.g. `.css-9rkwk9.css-9rkwk9`) are what is actually doing the trick. To quote the genius named Harry Roberts (@csswizardry):

> […] you can chain a selector with itself to increase its specificity.

(via https://csswizardry.com/2014/07/hacks-for-dealing-with-specificity/#safely-increasing-specificity)

Also see

* https://css-tricks.com/a-specificity-battle/#article-header-id-7
* https://stackoverflow.com/a/11581106

Reply:
So what about [previous comment](https://github.com/gatsbyjs/gatsby/pull/5813#discussion_r196211494) then?

> Seems like I should do the same here then?
> https://github.com/gatsbyjs/gatsby/pull/5813/files/b9c097e3eba19af0a824098aefe1e812ed00125b#diff-3f9ee2e9c0e318b6189a8f786092c9f5R164
> 
> `template-blog-post.js`
> 
> L164

(add `&&` selector to `template-blog-post.js` and move `color: `${colors.gatsby} !important`,` to it?)

Reply:
L164 seems to work fine without the `!important`: 
![image](https://user-images.githubusercontent.com/21834/41932602-a9ac6bea-7981-11e8-8fcd-8ec1977cbba5.png)

Reply:
Generally we should avoid (and have avoided—I'm quite sure I'm guilty here as I wasn't aware of `&&` when I first started using Glamor ;-)) using `!important`(see e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception).

If you need to override styles try if Glamor's `&&` will do and make `!important` a last resort.